### DISCUSSION TITLE
Differential Expression Calculations

### DISCUSSION THREAD

**Discussion by BjoernZiehr** [Nesvilab/FragPipe-Analyst-doc#26] [Q&A]
Hello, I am confused about some results from a recent LFQ analysis I conducted. There are samples that have zero "spectral counts" but they have "Intensity" values. I have three replicates, experimental conditions A and B. In one example, there are zero Spectral counts for all 3 replicates of condition A, but there are 3 "intensity" values. I cannot find these proteins or PSMs when using the PDV viewer, so I cannot validate. 
There are ~30 such cases in my proteinID list of ~3000. 

Additionally, for Differential Expression, I am confused. Using the case above, in my negative control "ctrl", there are zero intensity values in 3 replicates. However, in the DE, there is a log(fold change) given. 

log2(fold change) | -2.74        |        -2.2 |       -0.538
for                           A vs Ctrl      A vs B        B vs Ctrl

The mean intensities for B are 8.4e6, for A are 3.5e7. For ctrl: zero. Fold change is ~4, so the calculate fold change for A vs B is accurate, but it isn't possible for the ctrl comparisons. 

Any clarification would be appreciated.

**Comment by hsiaoyi0504** [comment]
@BjoernZiehr Sorry for taking long to see this. But can you clarify what kind of settings you are using? For example, did you choose intensity here in the analysis mentioned above? It sounds like it's NA and get imputed.

**Comment by hsiaoyi0504** [comment]
Closed due to no response.
